server/controllers/cv-controller.js

```diff
@@ -0,0 +1,44 @@
+const client = require('../lib/github-client');
+
+console.log(client.getUserProfile());
+function fetchingData(username, res) {
+	const userProfile = client.getUserProfile(username);
+	const PullRequest = client.getUserPullRequests(username);
+	return Promise.all([userProfile, PullRequest])
+  .then((data
+  	) =>{
+  	console.log(data);
+    return res.render('index', {code:data[0], codes:data[1][0]});
+  }).catch(err => {
+    console.log('Not retriving Data');
+  })
+}
+
+module.exports = (req, res) => {
+	const name = req.params.username;
+	return fetchingData(name, res);
```

Comment:
It's a little bit odd to have a function called `fetchingData`... but then you also pass it the `res` object so it both fetches _and_ sends the data to the user. Instead you could just pass the name to your fetchingData function, and then when the promise it returns resolves with a value you can pass that value to `res.render()` in here. 

Think back to the fruit game the other week - your controller, which deals directly with eth `req` and `res` objects should just sit there, firing off requests for information to other bits of your code or other systems, and when those requests respond with the data you want the controller is responsible for combining the data and sending it back to the user. If you start passing the `res` object around things can get pretty tangled  - what if you send it to 2 different bits of code... what if both those bits of code try to call `res.render()`... you'll end up with errors. So best to not pass it around

Do you even need the fetchingData function? Could you just do more work inside the controller function?